internal/services/network/private_endpoint_resource.go

```diff
@@ -753,7 +760,7 @@ func expandPrivateEndpointIPConfigurations(input []interface{}) *[]network.Priva
 			PrivateEndpointIPConfigurationProperties: &network.PrivateEndpointIPConfigurationProperties{
 				PrivateIPAddress: utils.String(privateIPAddress),
 				GroupID:          utils.String(subResourceName),
-				MemberName:       utils.String(subResourceName),
+				MemberName:       utils.String(v["member_name"].(string)),
```

Comment:
this is a breaking change for existing users/resourceS?

Reply:
i updated the code to not break current logic